### ISSUE TITLE
Error during "from asammdf import MDF"

### ISSUE THREAD

**Issue by mdiego1595** [danielhrisca/asammdf#346]
# Python version
_Please run the following snippet and write the output here_
```python
import platform
import sys
from pprint import pprint

pprint("python=" + sys.version)
pprint("os=" + platform.platform())

try:
    import numpy
    pprint("numpy=" + numpy.__version__)
except ImportError:
    pass

try:
    import asammdf
    pprint("asammdf=" + asammdf.__version__)
except ImportError:
    pass
```
# Code 

  ## MDF version
  _please write here the file version (you can run ``print(MDF(file).version)``)

  ## Code snippet
  _please write here the code snippet that triggers the error_
  
  ## Traceback
  _please write here the error traceback_
  
# Description

The fastest way to debug is to have the original file. For data protection you can use the static 
method _scramble_ to scramble all text blocks, and send the scrambled file by e-mail.

Hi,
I installed the asammdf library in python and when I try to start a script that uses the library, these errors occurs:

Traceback (most recent call last):
  File "C:\Users\Diego\Desktop\script\FileScriptCANalyzer.py", line 1, in <module>
    from asammdf import MDF
  File "C:\Users\Diego\AppData\Local\Programs\Python\Python37\lib\site-packages\asammdf\__init__.py", line 15, in <module>
    from .blocks.mdf_v2 import MDF2
  File "C:\Users\Diego\AppData\Local\Programs\Python\Python37\lib\site-packages\asammdf\blocks\mdf_v2.py", line 4, in <module>
    from .mdf_v3 import MDF3
  File "C:\Users\Diego\AppData\Local\Programs\Python\Python37\lib\site-packages\asammdf\blocks\mdf_v3.py", line 41, in <module>
    from pandas import DataFrame
  File "C:\Users\Diego\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\__init__.py", line 55, in <module>
    from pandas.core.api import (
  File "C:\Users\Diego\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\core\api.py", line 29, in <module>
    from pandas.core.groupby import Grouper, NamedAgg
  File "C:\Users\Diego\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\core\groupby\__init__.py", line 1, in <module>
    from pandas.core.groupby.generic import DataFrameGroupBy, NamedAgg, SeriesGroupBy
  File "C:\Users\Diego\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\core\groupby\generic.py", line 60, in <module>
    from pandas.core.frame import DataFrame
  File "C:\Users\Diego\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\core\frame.py", line 124, in <module>
    from pandas.core.series import Series
  File "C:\Users\Diego\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\core\series.py", line 4572, in <module>
    Series._add_series_or_dataframe_operations()
  File "C:\Users\Diego\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\core\generic.py", line 10349, in _add_series_or_dataframe_operations
    from pandas.core.window import EWM, Expanding, Rolling, Window
  File "C:\Users\Diego\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\core\window\__init__.py", line 1, in <module>
    from pandas.core.window.ewm import EWM  # noqa:F401
  File "C:\Users\Diego\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\core\window\ewm.py", line 5, in <module>
    import pandas._libs.window.aggregations as window_aggregations
ImportError: DLL load failed: Impossibile trovare il modulo specificato.

How to fix the issue?

Thank you

**Comment by danielhrisca** [comment]
@mdiego1595 you have a problem with your pandas installation. 

If you don't use a Python distribution (anaconda, winpython) you could try to install it again

```python
pip install -I pandas
```

**Comment by mdiego1595** [comment]
Thank you Daniel,
I tried with pip install -I pandas but this error occurs:
ERROR: Could not install packages due to an EnvironmentError: [WinError 5] Accesso negato: 'c:\\users\\diego\\appdata\\local\\programs\\python\\python37\\Lib\\site-packages\\numpy\\.libs\\libopenblas.PYQHXLVVQ7VESDPUVUADXEVJOBGHJPAY.gfortran-win_amd64.dll'
Consider using the `--user` option or check the permissions.

This error occurs also running prompt as administrator.

How can I fix? 
thank you in advance

**Comment by danielhrisca** [comment]
Close all python processes and the IDE

**Comment by mdiego1595** [comment]
ok I closed the IDLE and the command "pip install -I pandas" now runs but the problem still remain.
Any other idea?

**Comment by danielhrisca** [comment]
You can try to use the WinPyhton distribution

**Comment by mdiego1595** [comment]
I tried with WinPython distribution but now this error occurs:
C:\Users\Diego\Desktop\WPy64-3770\scripts>asammdf
Traceback (most recent call last):
  File "C:\Users\Diego\Desktop\WPy64-3770\python-3.7.7.amd64\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\Diego\Desktop\WPy64-3770\python-3.7.7.amd64\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\Diego\Desktop\WPy64-3770\python-3.7.7.amd64\Scripts\asammdf.exe\__main__.py", line 4, in <module>
  File "C:\Users\Diego\Desktop\WPy64-3770\python-3.7.7.amd64\lib\site-packages\asammdf\__init__.py", line 15, in <module>
    from .blocks.mdf_v2 import MDF2
  File "C:\Users\Diego\Desktop\WPy64-3770\python-3.7.7.amd64\lib\site-packages\asammdf\blocks\mdf_v2.py", line 4, in <module>
    from .mdf_v3 import MDF3
  File "C:\Users\Diego\Desktop\WPy64-3770\python-3.7.7.amd64\lib\site-packages\asammdf\blocks\mdf_v3.py", line 44, in <module>
    from ..signal import Signal
  File "C:\Users\Diego\Desktop\WPy64-3770\python-3.7.7.amd64\lib\site-packages\asammdf\signal.py", line 9, in <module>
    from .blocks.utils import MdfException, extract_cncomment_xml, SignalSource
  File "C:\Users\Diego\Desktop\WPy64-3770\python-3.7.7.amd64\lib\site-packages\asammdf\blocks\utils.py", line 27, in <module>
    from cchardet import detect
  File "C:\Users\Diego\Desktop\WPy64-3770\python-3.7.7.amd64\lib\site-packages\cchardet\__init__.py", line 1, in <module>
    from cchardet import _cchardet
ImportError: DLL load failed: Impossibile trovare il modulo specificato.

**Comment by mdiego1595** [comment]
I tried with other two PC but the last error remains.....

**Comment by danielhrisca** [comment]
![image](https://user-images.githubusercontent.com/20952040/78566665-403d3700-7828-11ea-8294-a022291ac17e.png)

in the path that you have used to unzip the WinPython distribution:
```
c:\WPy32-3741\python-3.7.4\python.exe -m pip install https://github.com/danielhrisca/asammdf/archive/development.zip PyQt5 pyqtgraph==0.11.0rc0
```

It will install the needed files

**Comment by mdiego1595** [comment]
I tried but the error still remains

**Comment by danielhrisca** [comment]
Open Spyder.exe from the WinPython folder and run your code from that IDE

**Comment by mdiego1595** [comment]
The DLL error remains also in Spyder.exe

**Comment by danielhrisca** [comment]
So you did 
```
C:\Users\Diego\Desktop\WPy64-3770\python-3.7.7.amd64\python.exe -m pip install cChardet
``` 

??

**Comment by mdiego1595** [comment]
Yes cChardet is installed but DLL error remains.

**Comment by danielhrisca** [comment]
Did you try to reinstall Windows? :D

I don't know what else I can say

**Comment by mdiego1595** [comment]
Ok is not a problem, but is strange that this error appears on three PC with different features (windows 10 32 bit, windows 10 64 bit and windows 7 SP1 32 bit).

**Comment by danielhrisca** [comment]
My bad. You should install cChardet 2.1.5

https://pypi.org/project/cchardet/2.1.5/

**Comment by mdiego1595** [comment]
The issue was fixed using cchardet 2.1.5 and pandas 1.0.1
Thank you
